tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# tasks file for ansible-playbook-telegraf
+#
+
+- name: fetch telegraf installation package
+  get_url: url=http://get.influxdb.org/telegraf/telegraf_{{ telegraf_version }}_amd64.deb dest=/tmp/telegraf_{{ telegraf_version }}_amd64.deb mode=0440
+
+- name: install telegraf package
+  command: dpkg --skip-same-version -i /tmp/telegraf_{{ telegraf_version }}_amd64.deb
```

Comment:
New commit https://github.com/alphagov/ansible-playbook-telegraf/commit/cc6ee5205d614af5ddf1e1c73bbe517d45a09d8b resolves this issue.